pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -866,7 +846,7 @@ func (l *LibvirtDomainManager) migrateHelper(vmi *v1.VirtualMachineInstance, opt
 	}
 
 	// initiate the live migration
-	dstURI := fmt.Sprintf("qemu+tcp://%s/system", net.JoinHostPort(ip.GetLoopbackAddress(), strconv.Itoa(LibvirtLocalConnectionPort)))
+	dstURI := fmt.Sprintf("qemu+unix:///system?socket=%s", migrationproxy.SourceUnixFile(l.virtShareDir, string(vmi.UID)))
```

Comment:
Since this is happening in `virt-launcher` we have to think about backward compatibility. There exist a few ways on how we could do that. One would for instance be bumping the grpc version and detect it via the negotiated version.

Reply:
Never mind. It is virt-launcher only. All good.